src/screens/ProfileScreen/elements/LinkAndSocialMedia.tsx

```diff
@@ -83,10 +83,7 @@ const InstagramButton = ({socialMessage}: {socialMessage: string}) => {
         };
         await Share.shareSingle(shareOptions);
       } else {
-        await Share.open({
-          url: 'https://www.instagram.com/direct/inbox/',
-          message: `${socialMessage}`
-        });
+        await Linking.openURL('https://www.instagram.com/direct/inbox/');
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from `Share.open` to `Linking.openURL` might cause a different user experience. The previous code was sharing a message along with opening the URL, but now it's just opening the URL without any message. If this is intentional and you want to open the URL directly in the default browser, then it's fine. However, if you still want to share a message, you should consider reverting to the old method or find an alternative way to share the message.
```diff
- await Linking.openURL('https://www.instagram.com/direct/inbox/');
+ await Share.open({
+   url: 'https://www.instagram.com/direct/inbox/',
+   message: `${socialMessage}`
+ });
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->